Android.mk

```diff
@@ -3,6 +3,9 @@ LOCAL_PATH := $(call my-dir)
 VERSION = $(shell git describe --always --dirty=-dev 2>/dev/null || echo Unknown)
 OPTEE_CLIENT_PATH ?= $(LOCAL_PATH)/../optee_client
 
+# Not sure why this isn't being set for me.
+TA_DEV_KIT_DIR := optee/optee_os/out/arm-plat-hikey/export-ta_arm64
```

Comment:
This will work fine:

$ make TA_DEV_KIT_DIR= optee/optee_os/out/arm-plat-hikey/export-ta_arm64